### ISSUE TITLE
7 Months in React - The Perilous Art of Swimming Upstream

### ISSUE THREAD

**Issue by stripethree** [KazooHR/engineering-lunchnlearn#33]
<a href="https://github.com/billfienberg"><img src="https://avatars3.githubusercontent.com/u/6130520?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [billfienberg](https://github.com/billfienberg)**
_Monday Jan 14, 2019 at 16:29 GMT_
_Originally opened as https://github.com/youearnedit/YEI-lunchnlearn/issues/28_

----

TLDR #1: Be wary of using React when your team is full of Ember devs because you think it will make hiring easier

TLDR #2: Be wary of relying on conventions when using unconventional technologies

https://youtu.be/rWH3UNIKFeg
